File: week6/Week6Day4/app/src/main/java/com/example/luisenriquez/week6day4/data/repositorymodule/Repository.java
package com.example.luisenriquez.week6day4.data.repositorymodule;

import android.arch.lifecycle.MutableLiveData;

import com.example.luisenriquez.week6day4.data.remote.models.MovieResponse;

public interface Repository
{
    MutableLiveData<MovieResponse> movieResponseLiveData = null;

    void getMovieResponse(String srchItm);
}

File: Reference/DeltaChallenge-master/DeltaChallenge-master/app/src/main/java/com/example/admin/deltachallenge/data/repository/Repository.kt
package com.example.admin.deltachallenge.data.repository

import android.arch.lifecycle.MutableLiveData
import com.example.admin.deltachallenge.data.models.NumbersResponse

interface Repository {

    val randomNumbersLiveData: MutableLiveData<NumbersResponse>

    val averageLiveData: MutableLiveData<Double>

    fun getRandomNumbers(): MutableLiveData<NumbersResponse>
}